README.md

```diff
@@ -362,10 +362,10 @@ max-globs = 100
 fail-on-max-globs = false
 
 # Maximum metrics could be returned by glob/wildcard in find request (currently
-# works only for trie index)
+# works only for trie index) (Default: 10,000,000)
 max-metrics-globbed  = 30000
```

Comment:
Maybe we should change values in README too?

Reply:
Hi @deniszh thanks for the feedback. 

The high default for `max-metrics-globbed` and `max-metrics-rendered` practically means no limit.

I was thinking about keeping the current value in readme and example as suggested values.

The new default are for existing go-carbon users where they usually don't have the config on their production system so that they aren't surprised by the new throttling control.

What do you think of it?

Reply:
Hi @deniszh I assume you are ok with keeping the value in readme and conf example files. ;)

But if needed, I can make a separate MR to alter them to the default value.